build.gradle

```diff
@@ -11,16 +11,20 @@ buildscript {
         classpath 'com.palantir.baseline:gradle-baseline-java:0.51.1'
         classpath 'com.palantir.gradle.gitversion:gradle-git-version:0.11.0'
         classpath 'gradle.plugin.org.inferred:gradle-processors:2.1.0'
+        classpath 'com.palantir.gradle.consistentversions:gradle-consistent-versions:0.17.0'
     }
 }
 
 apply plugin: 'com.palantir.baseline'
 apply plugin: 'com.palantir.git-version'
+apply plugin: 'com.palantir.consistent-versions'
 
 allprojects {
     version gitVersion()
     group 'com.palantir.conjure.java.runtime'
 
+    apply plugin: 'nebula.dependency-recommender' // necessary only to construct the BOM
```

Comment:
As a follow-up, we might be able to delete the bom publishing entirely as I think it's unused!